src/MCPClient/lib/clientScripts/create_mets_v2.py

```diff
@@ -36,12 +37,13 @@
 from uuid import uuid4
 
 import django
+import metsrw
 import scandir
 import six
 
 django.setup()
 # dashboard
-from django.utils import timezone
```

Comment:
Why is `timezone.now()` not relevant anymore?

Reply:
It was used to add the CREATEDATE attribute to the METS header. Now, with the inclusion of the metsrw on the final steps, if that attribute is set when the METS is parsed, metsrw adds the LASTMODDATE attribute thinking it's a re-ingest.

I removed that at some point and then introduced `datetime.datetime` for the new dates, to keep consistency with the metsrw.